lcls_tools/common/pydm_tools/displayUtils.py

```diff
@@ -1,17 +1,49 @@
 from functools import partial
 
-from PyQt5.QtWidgets import QMessageBox, QWidget
+from PyQt5.QtCore import QObject, pyqtSignal
+from PyQt5.QtWidgets import QLabel, QMessageBox, QWidget
 from qtpy.QtCore import Slot
 
+ABORT_STYLESHEET = "color: rgb(128, 0, 2);"
+FINISHED_STYLESHEET = "color: rgb(16, 128, 1);"
+STATUS_STYLESHEET = "color: rgb(7, 64, 128);"
+
+
+class WorkerSignals(QObject):
+    status = pyqtSignal(str)
+    finished = pyqtSignal(str)
+    error = pyqtSignal(str)
+    abort = pyqtSignal(bool)
+    
+    def __init__(self, label: QLabel = None):
+        super().__init__()
+        if label:
+            self.status.connect(label.setText)
+            self.status.connect(partial(label.setStyleSheet, STATUS_STYLESHEET))
+            
+            self.finished.connect(label.setText)
+            self.finished.connect(partial(label.setStyleSheet, FINISHED_STYLESHEET))
+            
+            self.error.connect(label.setText)
+            self.error.connect(partial(label.setStyleSheet, ABORT_STYLESHEET))
```

Comment:
Is this using the abort stylesheet on purpose? Or is it a copy/paste error?

Reply:
Maybe I should renamed it to `ERROR_STYLESHEET`... An abort felt like a type of error?